Mage.Tests/src/test/java/org/mage/test/cards/single/dis/AzoriusAethermageTest.java

```diff
@@ -0,0 +1,52 @@
+package org.mage.test.cards.single.dis;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+public class AzoriusAethermageTest extends CardTestPlayerBase {
+
+    /**
+     * Whenever you bounce a permanent (tokens included) you may pay {1}, if you do, draw a card
+     */
+    @Test
+    public void testBouncedLand() {
+        addCard(Zone.BATTLEFIELD, playerA, "Mountain", 3); // Used for paying ability cost
+        addCard(Zone.BATTLEFIELD, playerA, "Island", 2*3);
+        addCard(Zone.BATTLEFIELD, playerA, "Toggo, Goblin Weaponsmith");
+        addCard(Zone.HAND, playerA, "Boomerang",1);
+
+        // Permanents to bounce
+        addCard(Zone.HAND, playerA, "Plains", 1);
+        addCard(Zone.BATTLEFIELD, playerA, "Azorius Aethermage");
+        playLand(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Plains");    // Create a Rock token with Toggo
+        waitStackResolved(1, PhaseStep.PRECOMBAT_MAIN);
+
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Boomerang", "Plains");
+        waitStackResolved(1, PhaseStep.PRECOMBAT_MAIN, playerA, true);
+        setChoice(playerA, true);
```

Comment:
These tests don't work since I can't figure out how to say "yes" to Azorius Aethermage's triggered ability.

Reply:
[[ Azorius Aethermage]]

Reply:
Try something like:
setChoice(playerA, "Yes");

Reply:
I tried that, it doesn't work. Also the documentation of `setChoice` says they're interchangeable.

Reply:
Check the attached test file for Azorius Aethermage.  Works fine for me.
[AzoriusAethermageTest.txt](https://github.com/magefree/mage/files/8246636/AzoriusAethermageTest.txt)